### DISCUSSION TITLE
Shift Click Equipment Tele

### DISCUSSION THREAD

**Discussion by SomeRandoGamer** [runelite/runelite#15236] [Ideas]
Shift click teleport out side of the wilderness on useful teleports.

For Example: the Rada's blessing - while equipped I can hit shift and teleport to mount Karuulm,
 
Make no sense that you need to take it off to tele, especially after you play counting simulator at hydra, then need to bank. you can do the same action in the inventory just taking some steps out for QOL.

other useful teleports would be Karamja gloves, Morytaina legs, Explorers ring, Ardougne cloak,  Xerics talisman ( great for irons to bank ), games neck ect.

**Comment by EnvyRS** [comment]
If you hold shift and right click, you can change the left clicks and the shift left click options already my dude
For unequipped items, you can right click inventory icon and change the left and shift left clicks as well

**Reply by SomeRandoGamer** [reply]
hmm I cant seem to get it to work, is there a setting you need to enable?
 I can get it to work in the inventory as normal but when its equipped it wont swap from remove even when shift click is set.

**Comment by RsFoe** [comment]
turn off the 'no examine' plugin and then do it. It was thought to be fixed but apparently that part was somehow missed. Once you make the swaps you can safely turn it back on.

edit: the changes have been submitted just not pushed yet*

**Reply by SomeRandoGamer** [reply]
Holly shit boom!
your a legend wish I knew this before making this post and wasting everyone's time. thanks <3
